javascript/javascript-8-regex.md

```diff
@@ -52,8 +47,12 @@ About 2 hours
 - The pattern goes between the two forward slashes, and you can use back slashes to escape special characters
 - There are a lot of complex RegEx patterns. We'll teach the basic ones, but there are many more you can learn as you go. 
 
-### Demonstration
-Instructor demonstrates in the video walk through how to work with Regular Expressions in JavaScript.
+### Supplemental Materials
```

Comment:
actually, in this case please leave this section where it was.

Reply:
Done